buffer-env.el

```diff
@@ -68,6 +68,8 @@
     ("guix.scm" . "guix shell -D -f \"$0\" -- env -0")
     ("flake.nix" . "nix develop -c env -0")
     ("shell.nix" . "nix-shell \"$0\" --run \"env -0\"")
+    ("*.ps1" . "powershell -c '& { param($script) . $script > $null; Get-ChildItem env: |\
+                % {\"$($_.Name)=$($_.Value)`0\"} | Write-Host -NoNewLine } '")
```

Comment:
Hi @faerryn , can you show me how to change this command so that the script name is read from the second positional parameter? So what before was `powershell -c '...' .envrc` now becomes `powershell -c '...' <ignored> .envrc`?

Reply:
I think if you change the arglist `param($script)` to `param($ignored, $script)` then that should work, but I am really not sure.  Powershell is weird like that.

Reply:
Okay, thanks!